Gruntfile.js

```diff
@@ -174,7 +174,6 @@ module.exports = function (grunt) {
         // used by watch to copy shared/js to build dir
         exec: {
             copyjs: `cp shared/js/*.js build/${browser}/${buildType}/js/ && rm build/${browser}/${buildType}/js/*.es6.js`,
-            copyPolyFill: `cp node_modules/webextension-polyfill/dist/browser-polyfill.js build/${browser}/${buildType}/public/js/`,
```

Comment:
we don't have to manually copy it over because it's automatically bundled now that we added `import`s ?

Reply:
Yep :)